packages/web/src/pages/track-page/components/mobile/TrackHeader.tsx

```diff
@@ -318,6 +325,9 @@ const TrackHeader = ({
     goToRepostsPage(trackId)
   }, [goToRepostsPage, trackId])
 
+  const InnerImageElement = isSsrEnabled ? StaticImage : DynamicImage
```

Comment:
Since we will probably need this in a lot of instances, maybe we update dynamic image to handle a "src" prop and do something static with it?